index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Good job on implementing most of the changes provided by the previous reviewer but there are still `eslint-disable` line in the `index.js` page. Kindly remove this line as this defies the purpose of using linters in the first place and try to fix the error manually.
### Hint: 
I tried to remove it and I didn't find any linters errors.